bin/public/installNrpe.py

```diff
@@ -101,7 +101,11 @@ def _install_nrpe_plugins():
     """Install NRPE-plugins (to be executed remoteley) and SELinux-rules."""
     # Install packages and their dependencies.
     _install_nrpe_plugins_dependencies()
-    x("cp -p {0}lib/nagios/plugins_nrpe/* {1}".format(constant.SYCO_PATH, PLG_PATH))
+    host_config = config.host(net.get_hostname())
+    if host_config.is_guest():
```

Comment:
For plugin installation I don't think we need to split between physical others since the scripts are usually very small. Having all scripts on all machines is fine as long as we don't unnecessarily install a bunch of yum packages etc... I don't think this is worth the complexity it introduces..